pkg/lang/python/imports_test.go

```diff
@@ -298,48 +307,74 @@ print(other.hello_world)
 			name: "import full module with alias",
 			input: map[string]string{
 				"main.py": `
-import other as some_other
-some_other.hello_world()
-`,
+		import other as some_other
+		some_other.hello_world()
+		`,
 				"other.py": `pass`,
 			},
 			expect: map[string]core.Imported{
 				"main.py": map[string]core.References{
-					"other.py": NewSet[string]("hello_world"),
+					"other.py": NewSet("hello_world"),
 				},
 				"other.py": map[string]core.References{},
 			},
 		},
 		{
-			expectFailureDueTo: "#492",
+			name: "import qualified module",
+			input: map[string]string{
+				"main.py":        `import other.hello`,
+				"other/hello.py": `pass`,
+			},
+			expect: map[string]core.Imported{
+				"main.py": map[string]core.References{
+					"other/hello.py": NewSet[string](),
+				},
+				"other/hello.py": map[string]core.References{},
+			},
+		},
+		{
+			name: "import qualified module with attribute",
+			input: map[string]string{
+				"main.py":        `from other.hello import a`,
+				"other/hello.py": `pass`,
+			},
+			expect: map[string]core.Imported{
+				"main.py": map[string]core.References{
+					"other/hello.py": NewSet("a"),
+				},
+				"other/hello.py": map[string]core.References{},
+			},
+		},
+		{
+			expectFailureDueTo: "#492", // TODO https://github.com/klothoplatform/klotho-history/issues/492
 			name:               "import qualified module and use method",
 			input: map[string]string{
 				"main.py": `
-import other.hello
-other.hello.say_hi()
-`,
+		import other.hello
+		other.hello.say_hi()
+		`,
 				"other/hello.py": `pass`,
 			},
 			expect: map[string]core.Imported{
 				"main.py": map[string]core.References{
-					"other/hello.py": NewSet[string]("say_hi"),
+					"other/hello.py": NewSet("say_hi"),
 				},
 				"other.py": map[string]core.References{},
 			},
 		},
 		{
-			expectFailureDueTo: "#492",
+			expectFailureDueTo: "#492",                                        // TODO https://github.com/klothoplatform/klotho-history/issues/492
```

Comment:
mionr: why so much &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; whitespace?

Reply:
So that it lines up with the comment in the next line. gofmt does that automatically /shrug
I'll put it in the preceding line